src/devices/machine/pit8253.cpp

```diff
@@ -87,6 +96,11 @@ void pit8253_device::device_add_mconfig(machine_config &config)
 	PIT_COUNTER(config, "counter2", 0);
 }
 
+void ps2_pit_device::device_add_mconfig(machine_config &config)
+{
+	pit8253_device::device_add_mconfig(config);
+	PIT_COUNTER(config, "counter3", 0);
```

Comment:
You can pass your device finder in place of the literal tag here – there’s less chance of it getting out-of-sync that way.